test/cutcallback.jl

```diff
@@ -0,0 +1,119 @@
+using MathOptInterface
+const MOI = MathOptInterface
+using JuMP
```

Comment:
This is not needed and should be removed.

All relevant types & methods are defined in MOI already, so JuMP is only for the end users.
Also, this makes the tests fail now, because JuMP is not even installed.

I wonder why this didn't fail before.

Reply:
I only now noticed that JuMP is used to create the models for the tests :see_no_evil:.
Solvers should not depend on JuMP, so I'm afraid that the models have to be created "by hand" using MOI.

Reply:
> I wonder why this didn't fail before.

Probably because one of the tests failed before. There are two files for the tests and only one of them uses JuMP.

I'll replace the JuMP calls with MOI calls then.

Reply:
I'm finished with the replacement now. On my machine all testcases pass. Let's see if they also do on CI.

Reply:
Thanks, looking good so far.